packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -4632,6 +4632,30 @@ describe('plugin-meetings', () => {
             EVENT_TRIGGERS.MEETING_BREAKOUTS_UPDATE
           );
         });
+
+        it('listens to the timing that user joined into breakout', async () => {
+          const mainLocusUrl = 'mainLocusUrl123';
+
+          meeting.meetingRequest.getLocusStatusByUrl = sinon.stub().returns(Promise.resolve());
+
+          await meeting.locusInfo.emit(
+            {function: 'test', file: 'test'},
+            'CONTROLS_JOIN_BREAKOUT_FROM_MAIN',
+            {mainLocusUrl}
+          );
+
+          assert.calledOnceWithExactly(meeting.meetingRequest.getLocusStatusByUrl, mainLocusUrl);
+          const error = {statusCode: 403};
```

Comment:
issue: There should really be a test for when the error isn't a 403 and when the error doesn't have a status code

Reply:
Done